tailwind.config.js

```diff
@@ -11,40 +11,81 @@ module.exports = {
   darkMode: false, // or 'media' or 'class'
   theme: {
     shadow: {
-      DEFAULT: '0px 6px 16px rgb(0 0 0 / 12%)',
-      sm: '0px 1px 2px rgb(0 0 0 / 8%), 0px 4px 12px rgb(0 0 0 / 5%)',
+      z1: 'rgb(145 158 171 / 24%) 0px 1px 2px 0px',
```

Comment:
Too much numbers from this point foward, tl;dr

Reply:
Just copied values from Minimals foundation.